integration-tests/ACKNOWLEDGEMENTS.md

```diff
@@ -0,0 +1,21 @@
+# Acknowledgements
```

Comment:
You could integrate this in README.md

Reply:
@zakkak Dunno...Would you find it better? I feel like the Acknowledgements belong more to the License / Copyright notice files than to the user manual.  I don't have a strong opinion on it though...

Reply:
Me neither, I have seen it both as a separate file and as a section in Readme.